test/SparqlAntlrParserTest.cpp

```diff
@@ -855,3 +855,46 @@ TEST(SparqlParser, GroupClause) {
                              IsExpressionGroupKey("COUNT(?baz)"));
   }
 }
+
+TEST(SparqlParser, ValuesClause) {
+  expectCompleteParse(
+      parseValuesClause("VALUES ?test { \"foo\" }", {}),
+      IsValues<vector<std::string>, vector<vector<std::string>>>(
+          {"?test"}, {{"\"foo\""}}));
```

Comment:
I think the easiest way to make the types clear would be a 
```
using Vars = std::vector<std::string>'
using Vals = std::vector<std::vector<std::string>>`
// ...
IsValues(Vars{"?test"}, Vals{{"\"foo\""}})));
```
Or... even simpler:

```
auto expectValuesParse = [](const string& input, vector<string> expectedVars, vector<vector<string>> expectedVals) {
  expectCompleteParse(parseValuesClause(input), IsValues(expectedVars, expectedVals));
};
```
And then use this lambda in all the positive test cases